### DISCUSSION TITLE
How is "persist" working in react-native?

### DISCUSSION THREAD

**Discussion by LucaGabi** [ctrlplusb/easy-peasy#768] [Q&A]
Hi, I'm new to react-native and I was wandering how dose "persist" work? Where is the data stored ?

Thanks.

**Comment by ctrlplusb** [comment]
I am just playing around with React Native myself. Utilising the `AsyncStorage` that the `react-native` libs export. I have noticed some issues though and plan to do an update to Easy Peasy to make this more seamless. I will try post a code example when I have it sorted.
